src/XamlStyler.dotUltimate/src/rider/main/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,35 @@
+<idea-plugin>
+  <id>xamlstyler.rider</id>
+  <name>XAML Styler</name>
+  <version>_PLACEHOLDER_</version>
+  <vendor url="https://github.com/Xavalon/XamlStyler">Xavalon</vendor>
+  <idea-version since-build="_PLACEHOLDER_" until-build="_PLACEHOLDER_" />
+  <depends>com.intellij.modules.rider</depends>
+  <category>Formatting</category>
+
+  <description>
+<![CDATA[
+<p>XAML Styler is an extension that formats XAML source code based on a set of styling rules. This tool can help you/your team maintain a better XAML coding style as well as a much better XAML readability.</p>
+]]>
+  </description>
+
+  <extensions defaultExtensionNs="com.intellij">
+    <applicationConfigurable groupId="tools"
+                             instance="xavalon.plugins.xamlstyler.dotultimate.XamlStylerOptionsPage" />
+  </extensions>
+
+  <project-components>
+    <component>
+      <implementation-class>xavalon.plugins.xamlstyler.dotultimate.XamlStylerComponent</implementation-class>
+    </component>
+  </project-components>
+
+  <actions>
+    <action id="XamlStyler.Reformat"
+            text="Format with XAML Styler"
```

Comment:
Is there already a "Format XAML" action? If so, I get it. Otherwise, for consistency with other extensions, we should update it to "Format XAML".

Reply:
Err... I see in your GIF now. Burying the primary format action doesn't feel great. Can there be two actions? "Format XAML" (current file) and "Format XAML Files" (with sub menu).

Reply:
The primary format action is invoked from the menu (or with `Ctrl+Alt+Enter`). XAML Styler is special now, surfaced with `Alt+Enter` as a context action.

Have looked into tying XAML Styler in the normal reformat action, but that seems a rabbit hole.